lms/djangoapps/ccx/migrations/0002_add_master_course_staff_in_ccx#10877.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+from ccx_keys.locator import CCXLocator
+
+from courseware.courses import get_course_by_id
+
+from lms.djangoapps.ccx.utils import add_master_course_staff_to_ccx
+from lms.djangoapps.ccx.models import CustomCourseForEdX
+
+
+def add_master_course_staff_to_ccx_for_existing_ccx(apps, schema_editor):
+    """
+    Add all staff and admin of master course to respective CCX(s).
+    """
+    list_ccx = CustomCourseForEdX.objects.all()
+    for ccx in list_ccx:
+        course = get_course_by_id(ccx.course_id)
```

Comment:
I don't think you need to do this: the CCX model has a `course` method that should return the course